src/protocols/ethereum/components/TransferSendForm.vue

```diff
@@ -125,8 +123,32 @@ export default defineComponent({
     const route = useRoute();
     const { t } = useI18n();
     const { activeNetwork } = useNetworks();
+    const { marketData } = useCurrencies();
     const { balance } = useBalances();
-    const { getBaseFee } = useEthBaseFee();
+    const {
+      fee,
+      feeList,
+      feeSelectedIndex,
+      maxFeePerGas,
+      maxPriorityFeePerGas,
+      updateFeeList,
+    } = useEthFeeCalculation();
+    const { getProtocolAvailableTokens } = useFungibleTokens();
+
+    const ethTokensAvailable = computed(() => getProtocolAvailableTokens(PROTOCOLS.ethereum));
+
+    function getSelectedAssetValue(tokenContractId?: string, selectedAsset?: IAsset) {
+      const ethCoin = ProtocolAdapterFactory
+        .getAdapter(PROTOCOLS.ethereum)
+        .getDefaultCoin(marketData.value!, +balance.value);
+
+      if (tokenContractId) {
+        return ethTokensAvailable.value[tokenContractId as keyof AssetList] ?? ethCoin;
```

Comment:
No need to force the type if you will follow this comment: https://github.com/superhero-com/superhero-wallet/pull/2545/files#r1420120459